TelemetryManager/Tests/Mocks/Inc/MockMavlinkTranslator.hpp

```diff
@@ -0,0 +1,10 @@
+#include "gmock/gmock.h"
+#include "MavlinkTranslator.hpp"
+#include "TMCircularBuffer.hpp"
+#include "Official_Mavlink_2_Library/common/mavlink.h"
+
+class MockMavlinkTranslator : MavlinkTranslator
+{
+    MOCK_METHOD(void, bytesToMavlinkMsg, (TMCircularBuffer buffer));
+    MOCK_METHOD(void, addMavlinkMsgToByteQueue, (mavlink_message_t msg, TMCircularBuffer buffer));
+};
```

Comment:
Adding one empty line to the bottom of a file can help with passing the GCC compiling. https://www.unix.com/programming/137233-why-required-leave-empty-line-end-c-program.html